docs/configuration/configuration.md

```diff
@@ -550,6 +550,12 @@ subscription_id: <string>
 # instead be specified in the relabeling rule.
 [ port: <int> | default = 80 ]
 
+# If greater than 0s we will cache the Network Interace information 
+# Cache will be refresh every (cache_refresh_interval + refresh_interval + random(20s)).
+# For example for cache_refresh_interval = 20s , refresh_interval = 30s then cache will be refresh every between (50s - 70s).
+# By default the cache is disabled
+[ cache_refresh_interval: <duration> | default = 0s ]
```

Comment:
can we remove this parameter and use 5* refresh interval? to start with?

Reply:
Once issue with doing that is the cache will be enable by default and people will not be able to opt-in or out.

Reply:
Let's have it enabled by default. I dont think you can change the internal IP of a vm.

Reply:
Added the flag enabled by default but let the option is people want to opt out/in.
If you want me to remove the flag fully let me know.

Reply:
I think we can remove the flag and go for 10* refresh interval, then see how it behaves

Reply:
Ok, so people will not be able to disabled it. Just making that clear. 

I will remove the flag and use 10 time the refresh interval.

Reply:
removed